File: Server/src/main/content/global/skill/production/cooking/recipe/potato/impl/ChilliPotato.java
package content.global.skill.production.cooking.recipe.potato.impl;

import content.global.skill.production.cooking.recipe.potato.PotatoRecipe;
import core.game.node.item.Item;

/**
 * The Chilli potato.
 */
public class ChilliPotato extends PotatoRecipe {

    private static final Item CHILLI_POTATO = new Item(7054);

    private static final Item CHILLI_CON_CARNE = new Item(7062);

    @Override
    public Item getProduct() {
        return CHILLI_POTATO;
    }

    @Override
    public Item[] getIngredients() {
        return new Item[]{CHILLI_CON_CARNE};
    }

    @Override
    public boolean isTopping() {
        return true;
    }

    @Override
    public int getLevel() {
        return 41;
    }

    @Override
    public double getExperience() {
        return 10;
    }

}

File: Emulator/src/main/content/global/skill/cooking/potato/impl/ChilliPotato.kt
package content.global.skill.cooking.potato.impl

import content.global.skill.cooking.potato.PotatoRecipe
import org.rs.consts.Items
import core.game.node.item.Item

/**
 * Chilli potato.
 */
class ChilliPotato : PotatoRecipe() {

    override fun getProduct(): Item {
        return CHILLI_POTATO
    }

    override fun getIngredients(): Array<Item> {
        return arrayOf(CHILLI_CON_CARNE)
    }

    override fun isTopping(): Boolean {
        return true
    }

    override fun getLevel(): Int {
        return 41
    }

    override fun getExperience(): Double {
        return 10.0
    }

    companion object {
        private val CHILLI_POTATO = Item(Items.CHILLI_POTATO_7054)
        private val CHILLI_CON_CARNE = Item(Items.CHILLI_CON_CARNE_7062)
    }
}
